pkg/trace/exporter.go

```diff
@@ -0,0 +1,41 @@
+package trace
+
+import (
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/jaeger"
+	"go.opentelemetry.io/otel/sdk/resource"
+	"go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.17.0"
+)
+
+// NewExporter returns a Jaeger Exporter
+func newExporter(url string) (*jaeger.Exporter, error) {
+	endpoint := jaeger.WithEndpoint(url)
+	collector := jaeger.WithCollectorEndpoint(endpoint)
+	return jaeger.New(collector)
+}
+
+func newResource(version, seed string) *resource.Resource {
+	attrs := []attribute.KeyValue{
+		semconv.ServiceNameKey.String("authorino"),
+		semconv.ServiceVersionKey.String(version),
+	}
+	if seed != "" {
+		attrs = append(attrs, attribute.String("seed", seed))
```

Comment:
Thinking again about this. Perhaps it could expect a comma-separated key=value list instead. As long as it's in the same release, it can be done in a separate iteration.